test/runtests.jl

```diff
@@ -22,16 +25,33 @@ s = stringmime("application/postscript", fig);
 m = match(r"%%BoundingBox: *([0-9]+) +([0-9]+) +([0-9]+) +([0-9]+)", s)
 @test m !== nothing
 boundingbox = map(s -> parse(Int, s), m.captures)
-info("got plot bounding box ", boundingbox)
+Compat.@info("got plot bounding box ", boundingbox)
 @test all([300, 200] .< boundingbox[3:4] - boundingbox[1:2] .< [450,350])
 
 c = get_cmap("viridis")
-a = linspace(0,1,5)
-rgba = pycall(pycall(PyPlot.ScalarMappable, PyObject, cmap=c,
-                     norm=PyPlot.Normalize01)["to_rgba"], PyArray, a)
-@test rgba ≈ [ 0.267004  0.004874  0.329415  1.0
-               0.229739  0.322361  0.545706  1.0
-               0.127568  0.566949  0.550556  1.0
-               0.369214  0.788888  0.382914  1.0
-               0.993248  0.906157  0.143936  1.0 ]
+if VERSION < v"0.7.0"
+	a = linspace(0,1,5)
+else
+	a = range(0,step=1,length=5)
```

Comment:
can't we just call `Compat.range` in all versions?

Reply:
The `range` call is not correct.  The analogue of the linspace call is `range(0, stop=1, length=5)`

Reply:
Actually we can just do `0.0:0.25:1.0` in all versions.

Reply:
agreed will be much cleaner